src/chromium/interoperability-with-cpp.md

```diff
@@ -19,9 +19,22 @@ See the [CXX tutorial][1] for a full example of using this.
 <details>
 
 Talk through the diagram. Explain that behind the scenes, this is doing
-just the same as you previously did --- but by programmatically ensuring that
-the C++ and Rust sides match, cxx can ensure there aren't obvious errors
-with object lifetimes, string lengths, etc. It reduces lots of fiddly
```

Comment:
I retained the spirit of "ensuring that the C++ and Rust sides match".

I dropped "ensure there aren't obvious errors with lifetimes".  I am not aware of how `cxx` helps to avoid such errors.  Please shout if I should retain this.

I kind of, sort of retained "can ensure there aren't obvious errors with [...] string lengths" by talking about slice/span interop.

Reply:
I like your revised list, but please put back the two points you removed. Perhaps they're more clearly worded like this:

* Patterns are provided for RAII ownership of Rust objects in C++ and vice-versa, without error-prone manual deletion.
* String lengths and pointers are kept synchronized, and differing expectations about Rust and C++ strings (in particular, UTF8 validity) are encoded in the type system so mistakes can't be made.

Reply:
Done, although I've reworded the suggestions and phrased them as examples of core-types that the automation supports.